tests/test_template.py

```diff
@@ -38,3 +40,21 @@ def test_find_shapes_from_one_slide(template):
 def test_find_shapes_index_out_of_range(template):
     with pytest.raises(IndexError):
         shapes = template._find_shapes(0, 'logo')
+
+
+def test_save_pdf(template):
+    try:
+        subprocess.Popen(['libreoffice', '--version'],
+                         stdout=subprocess.DEVNULL)  # check if libreoffice is installed
+
+        output_path = 'test/test.pdf'
+        template.save_pdf(output_path)
+        path = Path(output_path)
+        assert path.exists() == True
+        if path.exists():
+            os.remove(path)
+            Path.rmdir(path.parent)
+    except FileNotFoundError:
+        with pytest.raises(LibreOfficeNotFoundError):
```

Comment:
I would rather `pytest.skip()` if LibreOffice is not there.

Also, please limit the scope of this avialability test to the top call.

Reply:
ok.

Reply:
What do you mean by this "limit the scope of this avialability test to the top call." ?  Should I create special marker for this test method?

Reply:
Moving the `except` up immediately behind the availability check.